docs/user/custom-domains.rst

```diff
@@ -1,105 +1,57 @@
 Custom domains
 ==============
 
-You can serve your documentation project from your own domain,
+By configuring a *custom domain* for your project,
+your project can serve documentation from a domain you control,
 for instance ``docs.example.com``.
 This is great for maintaining a consistent brand for your product and its documentation.
 
 .. _default-subdomain:
 
 .. rubric:: Default subdomains
 
-*By default*, your documentation is served from a Read the Docs *subdomain* using the project's :term:`slug`:
+*Without a custom domain configured*,
+your project's documentation is served from a Read the Docs domain using a unique subdomain for your project:
 
-* ``<slug>.readthedocs.io`` for |org_brand|
-* ``<slug>.readthedocs-hosted.com`` for |com_brand|.
+* ``{project name}.readthedocs.io`` for |org_brand|.
+* ``{organization name}-{project name}.readthedocs-hosted.com`` for |com_brand|.
```

Comment:
We're using `<>` for the example text in the docs: https://dev.readthedocs.io/en/latest/style-guide.html#content

Reply:
```suggestion
* ``<project name>.readthedocs.io`` for |org_brand|.
* ``<organization name>-<project name>.readthedocs-hosted.com`` for |com_brand|.
```